### DISCUSSION TITLE
How can ignore SSL certificate in local env?

### DISCUSSION THREAD

**Discussion by apomili** [graphcommerce-org/graphcommerce#1668] [Ideas] [answered]
Hi, we use Magento 2 via Warden. Warden use self signed certificate.

I could'nt find a solution to bypass the error while running "yarn codegen"

In other next project seem adding NODE_TLS_REJECT_UNAUTHORIZED = 0  in .env file will do the dirty work but not in graphcommerce.
Could you please help me?

**Comment by paales** [comment, marked as answer]
Can you try adding `skipSSLValidation: true` to your `.meshrc.yml`?

**Reply by apomili** [reply]
It works, thx :)


Edit: but now I have this error

https://mage245.test/graphql?query=query+[...]http%3A%2F%2Fmage245.test%2Fgraphql%22%7D failed, reason: unable to verify the first certificate

This error happened while generating the page. Any console logs will be displayed in the terminal window.

**Reply by paales** [reply]
```bash
NODE_TLS_REJECT_UNAUTHORIZED=0 yarn codegen
NODE_TLS_REJECT_UNAUTHORIZED=0 yarn dev
```

**Reply by paales** [reply]
@apomili Does that work for you?

**Reply by apomili** [reply]
Hi paales, yes, seems to work.
I'm not completely sure because the command yarn dev rise new error on parse (Error: Unexpected token < in JSON at position 0). Since the project is clean I expected it to work (as it did with an online Magento), but it looks more like an application error than a communication error.

**Reply by paales** [reply]
Hmmm, can you try and remove node_modules and yarn.lock and try running `yarn` again to see if that fixes it?

It is related to this package, it might install multiple versions if this doesn't go right:

https://github.com/graphcommerce-org/graphcommerce/blob/f2b7bc855a4d93189227bb0ff08f1dee9763a428/examples/magento-graphcms/package.json#L104-L106

As a workaround if you can't get it to work, set this to false:

https://github.com/graphcommerce-org/graphcommerce/blob/f2b7bc855a4d93189227bb0ff08f1dee9763a428/examples/magento-graphcms/.meshrc.yml#L18

**Comment by sickdaflip** [comment]
NODE_TLS_REJECT_UNAUTHORIZED=0 yarn codegen
yarn run v1.22.11
$ mesh build && graphql-codegen
💡 🕸️  Mesh Cleaning existing artifacts
💡 🕸️  Mesh Reading the configuration
💡 🕸️  Mesh Generating the unified schema
(node:207852) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
(node:207852) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
💡 🕸️  Mesh Generating artifacts
💡 🕸️  Mesh Generating index file in TypeScript
💡 🕸️  Mesh Writing index.ts for CJS to the disk.
💡 🕸️  Mesh Cleanup
💡 🕸️  Mesh Done! => /var/customers/webs/commerce/my-project/.mesh
✔ Parse Configuration
✔ Parse Configuration
⚠ Generate outputs
  ✔ Generate node_modules/@graphcommerce/graphql/generated/types.ts
  ✔ Generate node_modules/@graphcommerce/graphql/generated/fragments.json
  ❯ Generate to . (using EXPERIMENTAL preset "@graphcommerce/graphql-codegen-near-operation-file")
    ✔ Load GraphQL schemas
    ✔ Load GraphQL documents
    ✖ Unable to find field "mollieProcessTransaction" on type "Mutation"!
error Command failed with exit code 1.

I just can not manage it :(

**Reply by paales** [reply]
@sickdaflip Can you do try step? https://www.graphcommerce.org/docs/getting-started/create#remove-unused-psps
